Documentation/ColumnsConfig/Type/Link/Index.rst

```diff
@@ -1,5 +1,6 @@
 .. include:: /Includes.rst.txt
 
+.. _columns-input-renderType-inputLink:
```

Comment:
should we not rather fix all areas that still have the old link, as they might very well describe the old renderType?

Reply:
I don't know. The type link is a direct replacement for renderType inputLink. We would just break some links, if we omit it.

Reply:
Ok, lets do it then